client/src/components/BlogItem/BlogItem.js

```diff
@@ -0,0 +1,35 @@
+import { Col, Row, Typography } from 'antd';
+import React from 'react';
+import { Link, useRouteMatch } from 'react-router-dom';
+import { Trans } from '@lingui/macro';
+
+const { Paragraph, Text, Title } = Typography;
+
+const BlogItem = ({ postDetails }) => {
+  const { url } = useRouteMatch();
+  const { image, title, created, text, slug } = postDetails;
+  const authorFullName = `${postDetails.author_first_name} ${postDetails.author_last_name}`;
+  return (
+    <Row className="blog-item" gutter={{ md: 24, lg: 32 }}>
+      <Col className="blog-image" sm={24} md={12} lg={10}>
+        <img src={image} alt={title} />
+      </Col>
+      <Col className="blog-meta" sm={24} md={12} lg={14}>
+        <Title level={3} underline>
+          {title}
+        </Title>
+        <Text type="secondary">
+          <Trans>
+            Published by {authorFullName} • {new Date(created).toLocaleDateString()}
+          </Trans>
+        </Text>
+        <Paragraph>{text.replace(/<.>|<\/.>/, '').slice(0, 200)}...</Paragraph>
```

Comment:
There seem to be 2 issues with the current implementation. 
1. Special characters aren't displayed (e.g. `ó` is changed to `&o`)
2. HTML tags are displayed and not hidden - we should see the first 200 chars without formatting 
(screenshot from [the preview deployment](https://seismic-risc-96dmyrkoj-code4romania.vercel.app/blog))
![image](https://user-images.githubusercontent.com/2820084/109382126-2cd70680-78e7-11eb-9f60-acd6e632b6c9.png)